server/log.js

```diff
@@ -0,0 +1,15 @@
+const conf = require('./config.js');
+
+let notLocalHost =
+  conf.env === 'production' &&
+  conf.s3_bucket !== 'localhost' &&
+  conf.bitly_key !== 'localhost';
```

Comment:
how about we put this in `config.js` and attach it to conf, so you'd use it like `conf.notLocalHost`?